build-dist.bat

```diff
@@ -0,0 +1,18 @@
+@echo off
```

Comment:
You should install git and use bash to run the shell script. Most devs work on linux or MacOS and I'd considered a batch file to be non-standard.